packages/contentstack-import/src/import/modules-js/environments.js

```diff
@@ -98,6 +94,15 @@ module.exports = class ImportEnvironments {
         })
         .catch(function (error) {
           writeFileSync(envFailsPath, self.fails);
+          let message_content_type = '';
+          if (error.request !== undefined && JSON.parse(error.request.data).environment !== undefined) {
+            if (JSON.parse(error.request.data).environment.name) {
+              message_content_type =
+                ' Failed environment: ' + JSON.parse(error.request.data).environment.name;
+            }
+            error.errorMessage = error.errorMessage + message_content_type;
+          }
+          log(self.config, formatError(error.errorMessage), 'error');
```

Comment:
@cs-raj Could you please check to see if we need this? It has already been addressed above.

Reply:
@aman19K  changes are done and removed the redundant code